x-pack/plugins/fleet/server/services/epm/archive/validation.ts

```diff
@@ -261,25 +350,28 @@ export function parseAndVerifyStreams(
           `Invalid manifest for data stream ${dataStreamPath}: stream is missing one or more fields of: input, title`
         );
       }
+
       const vars = parseAndVerifyVars(manifestVars, `data stream ${dataStreamPath}`);
 
+      const streamObject: RegistryStream = {
+        input,
+        title: streamTitle,
+        template_path: templatePath || 'stream.yml.hbs',
+      };
+
+      if (vars.length) {
+        streamObject.vars = vars;
+      }
+
       // default template path name see https://github.com/elastic/package-registry/blob/master/util/dataset.go#L143
```

Comment:
Is this comment still relevant right here or should it be moved up above?

Reply:
This just leads to a 404 so I'll opt to remove it.